examples/Manifest.toml

```diff
@@ -157,9 +157,9 @@ version = "0.1.2"
 
 [[deps.CLIMAParameters]]
 deps = ["DocStringExtensions", "TOML", "Test"]
-git-tree-sha1 = "8cb1d69b6b5fc41be48bea969a68763a17843f1e"
+git-tree-sha1 = "a769996f7ed7a2f1fb070c4837370603387e5eb6"
```

Comment:
We can make these changes in a separate `julia .dev/up_deps.jl` PR, since changes miss the other environments (e.g., docs, test, perf)

Reply:
I'll open one now

Reply:
Opened #1335

Reply:
The above PR got merged, So we should be able to rebase this one